tests/test_workers/test_tasks/test_opm_operations.py

```diff
@@ -376,6 +378,7 @@ def test_opm_registry_rm_fbc(
         tmpdir,
     )
 
+    mock_srt.assert_called_once()
```

Comment:
I would personally fill overwrite_from_index_token parameter, when calling opm_registry_rm_fbc(). This way it would be possible to test if we are passing the parameter to set_registry_token() - now If I understand the code right this feature is not tested at all. If we would fill the parameter, this line could be changed to mock_srt.assert_called_once_with(overwrite_from_index_token, from_index, append). 
But I am not sure if this set_registry_token can do something to the rest of the test.

Reply:
This piece is being tested in other parts of the code. But you're right, it doesn't hurt to have more aggressive checks. Changed :)